lte/gateway/c/core/oai/oai_mme/oai_mme.c

```diff
@@ -149,9 +150,9 @@ int main(int argc, char* argv[]) {
 #endif
   CHECK_INIT_RETURN(s1ap_mme_init(&mme_config));
 
-  if (mme_config.enable_converged_core) {
-    CHECK_INIT_RETURN(ngap_amf_init(&amf_config));
-  }
+  // if (mme_config.enable_converged_core) {
```

Comment:
why is this commented out?

Reply:
it was commented initial merge magma-5G, in latest code the above check was uncommented.

resolved, uncommented enable_converged_core check